src/supermarket/config/initializers/content_security_policy.rb

```diff
@@ -4,17 +4,17 @@
 # For further information see the following documentation
 # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
 
-# Rails.application.config.content_security_policy do |policy|
-#   policy.default_src :self, :https
-#   policy.font_src    :self, :https, :data
-#   policy.img_src     :self, :https, :data
-#   policy.object_src  :none
-#   policy.script_src  :self, :https
-#   policy.style_src   :self, :https
+Rails.application.config.content_security_policy do |policy|
+  policy.default_src :self, :https
+  policy.font_src    :self, :https, :data
+  policy.img_src     :self, :https, :data
+  policy.object_src  :none
+  policy.script_src  :self, :https
```

Comment:
Does this prevent us from loading the external analytics tags?

Reply:
This is a configuration setting, having this config presence in the application we can add/remove content security wherever we need in the application.

Reply:
Let's just make sure to test this once it hits staging so we can confirm all our external content loads as expected.